boa_wasm/Cargo.toml

```diff
@@ -13,7 +13,7 @@ rust-version = "1.56"
 
 [dependencies]
 Boa = { path = "../boa", features = ["console"] }
-wasm-bindgen = "0.2.79"
+wasm-bindgen = "=0.2.78"
```

Comment:
How come this is locked?

Reply:
Razican said it just above, there's a bug in 0.2.79 that causes CI to fail.